src/build.d

```diff
@@ -637,7 +610,9 @@ void processEnvironment()
     string[] warnings;
 
       // TODO: allow adding new flags from the environment
-    string[] dflags = ["-version=MARS", "-w", "-de", "-dip25", env["PIC_FLAG"], env["MODEL_FLAG"], "-J"~env["G"]];
+    string[] dflags = ["-version=MARS", "-w", "-de", env["PIC_FLAG"], env["MODEL_FLAG"], "-J"~env["G"]];
+    if (env["HOST_DMD_KIND"] != "gdc")
+        dflags ~= ["-dip25"]; // gdmd doesn't support -dip25
```

Comment:
I don't know which version of GDC you're supporting, but GDC does have a `-ftransition=dip1000` flag.

Reply:
For this particular mechanic I'm just copying what the Makefile is doing.  Changing the functionality seems like a good change for another PR.

Reply:
GDC 8 is the minimum.